packages/server/lib/controllers/sync.controller.ts

```diff
@@ -339,6 +339,11 @@ class SyncController {
     }
 
     public async triggerAction(req: Request, res: Response, next: NextFunction) {
+        const active = tracer.scope().active();
+        const span = tracer.startSpan('sync.triggerAction', {
+            childOf: active as Span
```

Comment:
I know I have done it before but I don't remember why specifying `childOf` is necessary. Isn't a span automatically started in the context of the active one?

Reply:
@bodinsamuel I'm unsure here, any idea?